pkgs/data/documentation/zeal/default.nix

```diff
@@ -1,38 +1,24 @@
-{ stdenv, fetchFromGitHub, pkgconfig, qt5, libarchive }:
+{ stdenv, fetchurl, pkgconfig, qt5, libarchive }:
 
 stdenv.mkDerivation rec {
-  version = "20141123";
+  version = "0.1.1";
   name = "zeal-${version}";
 
-  src = fetchFromGitHub {
-    owner = "zealdocs";
-    repo = "zeal";
-    rev = "76405f8387d6a82697faab9630c78f31417d8450";
-    sha256 = "1057py3j2flzxyiks031s0mwm9h82v033iqn5cq8sycmrb3ihj2s";
+  src = fetchurl {
```

Comment:
Please, use ``fetchzip`` or ``fetchFromGitHub``, because github's generated tarballs change hashes in time.